lib/axon_onnx/deserialize.ex

```diff
@@ -759,10 +759,20 @@ defmodule AxonOnnx.Deserialize do
         Map.put(used_params, output_name, %{"kernel" => kernel, "bias" => bias})
       end
 
-    updated_axon =
-      Map.put(
-        axon,
-        output_name,
+    conv =
+      if group > 1 do
+        Axon.depthwise_conv(
+          axon_inp,
+          units,
+          kernel_size: kernel_size,
+          feature_group_size: group,
+          kernel_dilation: dilations,
+          padding: padding_config,
+          strides: strides,
+          use_bias: maybe_bias != [],
+          name: output_name
```

Comment:
So I think this instead needs to be:

```suggestion
          axon_inp,
          group,
          kernel_size: kernel_size,
          kernel_dilation: dilations,
          padding: padding_config,
          strides: strides,
          use_bias: maybe_bias != [],
          name: output_name
```

`group` is the `channel_multiplier` or `depth_multiplier`